### ISSUE TITLE
Feature Request: change download folder

### ISSUE THREAD

**Issue by inferno-prime** [Arsenal21/simple-download-monitor#89]
Would it be possible to include a feature like this in the free version? 

Will you be adding more features in the future or will you be turning those ideas into paid plugins like other plugin authors? I'm asking because I really do enjoy using the free version.

**Comment by Arsenal21** [comment]
Thank you for the suggestion. This is something we actually have on our Wishlist for the free plugin.

**Comment by inferno-prime** [comment]
If you don't mind me asking. Do you plan to add any more features to the free version? because I could probably come up with some suggestions?

**Comment by Arsenal21** [comment]
Yes, over time we will of-course add features. The following should give you a good idea of our approach and thinking in general. You will see this reflected throughout most of our plugins.
---
Our primary objective is to keep the plugin lightweight, low maintenance, and always working with the latest version of WordPress.

We generally try to keep our plugins simple to use and lightweight so they do not slow down your site. Depending on the needs of your project, you may need a more complex plugin to fit your needs and we understand that our plugin will not fit everyone's needs.

Every feature or option that is added to the plugin has additional maintenance and support requirements. Just adding more options is not the whole equation. You will notice that many plugins out there offer something then later the plugin gets abandoned after a few years because they cannot keep up maintaining the plugin. So we always carefully evaluate things to try to understand how much usage a feature will get, how much complexity it will add to our codebase, how much maintenance weight it will add in the future etc. 

Sometimes we don't know how some of the external operators (API from external sources) will change stuff on their end, what restrictions they will impose and so on and so forth. When we don't have a clear view of it, we try to stay away from it (especially, if only a very small user base wants to use it). We also maintain a wish-list of features/things that users want, so we can evaluate what things are being asked a lot. Some features are also hard to add given that we want to keep maintaining a simple architecture for our plugin (some features will inherently add complexity to the architecture that will have unintended consequences down the road).

We have limited developer and support resources. We have been doing this plugin development and maintenance for 10+ years now and we intend to do this for many more years. So we try our best to make sure we are within our limits to be able to continue to maintain what we offer for a longer time horizon. Give less features but maintain whatever we give. We use most of our plugins ourselves on many of our other websites so just pushing products with a lot of features is not our main goal.

**Comment by inferno-prime** [comment]
Is there any possibility someone could request a feature and it actually being included?
